src/test/spec/unified_runner/operation.rs

```diff
@@ -37,15 +36,21 @@ use crate::{
     RUNTIME,
 };
 
-#[async_trait]
 pub trait TestOperation: Debug {
-    async fn execute_test_runner_operation(&self, test_runner: &mut TestRunner);
+    fn execute_test_runner_operation<'a>(
+        &'a self,
+        _test_runner: &'a mut TestRunner,
+    ) -> BoxFuture<'a, ()> {
```

Comment:
Does this (and likewise below) need a default body?  It didn't have one before.

Reply:
They didn't have them previously because default implementations didn't place nicely with `async_trait`, see this comment for more info: https://github.com/mongodb/mongo-rust-driver/blob/master/src/test/spec/v2_runner/operation.rs#L43.